modules/containers-signature-verify-skopeo.adoc

```diff
@@ -0,0 +1,70 @@
+//
+// * security/container_security/security-container-signature.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="containers-signature-verify-skopeo_{context}"]
+= Using skopeo to verify signatures of Red Hat container images
+
+You can verify the signatures for container images included in an OCP release image by pulling those signatures from link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/[ocp-release]. Because the signatures on the mirror site are not in a format readily understood by podman or CRI-O, you can use the `skopeo standalone-verify` command to verify that the your release images are signed by Red Hat.
+
+.Prerequisite
+* You have installed the `skopeo` command-line utility.
+
+.Procedure
+
+. Get the full SHA for your release by running the following command:
++
+[source,terminal]
+----
+$ oc admin release info <release_version>
+----
+Substitute <release_version> with your release number, for example, `4.12.23`.
++
+.Example output
++
+[source,terminal]
+----
+Pull From: quay.io/openshift-release-dev/ocp-release@sha256:2309578b68c5666dad62aed696f1f9d778ae1a089ee461060ba7b9514b7ca417 -
+----
+//Q: is this what we should see if we run the oc admin release info command?
+
+. Pull down the Red Hat release key by running the following command:
++
+[source,terminal]
+----
+$ curl -o pub.key https://access.redhat.com/security/data/fd431d51.txt
+----
+//Q: is there a user value to enter here that our admin needs to get from somewhere?
+
+. Get the signature file for the specific release that you want to verify by running the following command:
++
+[source,terminal]
+----
+$ curl -o signature-1 https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/sha256%3D<sha_from_version>/signature-1
+----
+Replace `<sha_from_version>` with the full link to the mirror site that matches the SHA of your release. For example, the link to the signature for the 4.12.23 release is `https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/sha256%3D2309578b68c5666dad62aed696f1f9d778ae1a089ee461060ba7b9514b7ca417/signature-1`.
+
+. Get the manifest for the release image by running the following command:
++
+[source,terminal]
+----
+$ skopeo inspect --raw docker://<quay_link_to_release> > manifest.json
+----
+Replace `<quay_link_to_release>` with the output of the `oc admin release info` command. For example, `quay.io/openshift-release-dev/ocp-release@sha256:2309578b68c5666dad62aed696f1f9d778ae1a089ee461060ba7b9514b7ca417`.
+
+. Use skopeo to verify the signature:
++
+[source,terminal]
+----
+$ skopeo standalone-verify manifest.json quay.io/openshift-release-dev/ocp-release:<release_number>-<arch> any signature-1 --public-key-file pub.key
+----
+Replace <release_number> with `4.12.23`, for example.
+Replace <arch> with your architecture, such as `x86_64`, for example.
++
+.Example output
+[source,terminal]
```

Comment:
Another one where I'm surprised the lack of `+` doesn't affect the rendering. I do wonder if this could impact the portal rendering:

```suggestion
+
[source,terminal]
```

Reply:
I have always based that guidance off of the examples in the repo guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#node-names (you need to scroll a little to find the example).

For this case, I think it works either way, but consistency helps avoid spurious merge conflicts.

I usually try to call it out, but sometimes I still miss it.